API.md

```diff
@@ -286,7 +286,7 @@ in the [API.md](https://github.com/containers/libpod/blob/master/API.md) file in
 <div style="background-color: #E8E8E8; padding: 15px; margin: 10px; border-radius: 10px;">
 
 method Attach(name: [string](https://godoc.org/builtin#string), detachKeys: [string](https://godoc.org/builtin#string), start: [bool](https://godoc.org/builtin#bool)) </div>
-Attach takes the name or ID of a container and sets up a the ability to remotely attach to its console. The start
+Attach takes the name or ID of a container and sets up the ability to remotely attach to its console. The start
```

Comment:
API.md is actually autogenerated from comments in the `cmd/podman/varlink/io.podman.varlink` via`make varlink_api_generate` so you'll need to make the changes there in order for them to persist.

Reply:
Thanks, I'll do necessary changes.